services/subnet_manager/src/test/java/com/futurewei/alcor/subnet/SubnetControllerTests.java

```diff
@@ -97,7 +97,7 @@ public void createSubnetState_create_pass () throws Exception {
                 .thenReturn(routeWebJson);
         Mockito.when(subnetService.allocateMacGateway(anyString(), anyString(), anyString()))
                 .thenReturn(macResponse);
-        Mockito.when(subnetService.allocateIPGateway(anyString()))
+        Mockito.when(subnetService.allocateIPGateway(anyString(), anyString()))
```

Comment:
Can we have a UT covering bad CIDR block case?

Reply:
Finished!